File: Problem_Solving/Algorithms/Warmup/3_Compare_The_Triplets/Solution.py
import math
import os
import random
import re
import sys

def compareTriplets(a, b):
    categories=len(a)
    aliceScore=0
    bobScore=0

    for i in range(categories):
        aRating=a[i]
        bRating=b[i]
        if aRating>bRating:
            aliceScore+=1
        elif aRating<bRating:
            bobScore+=1

    return [aliceScore,bobScore];

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    a = list(map(int, input().rstrip().split()))

    b = list(map(int, input().rstrip().split()))

    result = compareTriplets(a, b)

    fptr.write(' '.join(map(str, result)))
    fptr.write('\n')

    fptr.close()

File: Problem_Solving/Algorithms/Warmup/3_Compare_The_Triplets/Solution.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {

    static List<int> compareTriplets(List<int> a, List<int> b) {
        int numberOfCategories=a.Count;
        int aliceScore=0;
        int bobScore=0;

        for(int i=0;i<numberOfCategories;i++){
            int aRating=a[i];
            int bRating=b[i];
            if(aRating>bRating)
                aliceScore++;
            else if(aRating<bRating)
                bobScore++;
        }

        return new List<int>(){aliceScore,bobScore};
    }

    static void Main(string[] args) {
        TextWriter textWriter = new StreamWriter(@System.Environment.GetEnvironmentVariable("OUTPUT_PATH"), true);

        List<int> a = Console.ReadLine().TrimEnd().Split(' ').ToList().Select(aTemp => Convert.ToInt32(aTemp)).ToList();

        List<int> b = Console.ReadLine().TrimEnd().Split(' ').ToList().Select(bTemp => Convert.ToInt32(bTemp)).ToList();

        List<int> result = compareTriplets(a, b);

        textWriter.WriteLine(String.Join(" ", result));

        textWriter.Flush();
        textWriter.Close();
    }
}
